apiserver/facades/client/charms/client.go

```diff
@@ -33,8 +35,8 @@ import (
 type API struct {
 	*charmscommon.CharmsAPI
 	authorizer   facade.Authorizer
-	backendState BackendState
-	backendModel BackendModel
+	backendState interfaces.BackendState
```

Comment:
I know there is no need to do this but it would be cleaner to alias the import to something like `charminterfaces` so we know what is the purpose of the interface.